src/OrleansRuntime/Scheduler/ClosureWorkItem.cs

```diff
@@ -54,7 +55,7 @@ public override string ToString()
             var detailedName = nameGetter != null ? "" :  // if NameGetter != null, base.ToString() will print its name.
                 String.Format(": {0}->{1}", 
                     (continuation.Target == null) ? "" : continuation.Target.ToString(),
-                    (continuation.Method == null) ? "" : continuation.Method.ToString());
+                    (continuation.GetMethodInfo() == null) ? "" : continuation.GetMethodInfo().ToString());
```

Comment:
can you please change it to call `GetMethodInfo()` only once?
I know its unrelated to this PR, but would be nice to optimize if we are already here.